### PR TITLE
epkowa: add support for 'Perfection V37/V370' scanners

### PR DESCRIPTION
**Pull request by nek0**
###### Motivation for this change

Scanner support for the model I own was missing.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/drivers/epkowa/default.nix [hunk 1]
```diff
@@ -65,6 +65,40 @@ let plugins = {
...
+    };
+
+    nativeBuildInputs = [ autoPatchelfHook rpm ];
+
+    installPhase = ''
+      cd plugins
+      ${rpm}/bin/rpm2cpio iscan-plugin-perfection-v370-*.x86_64.rpm | ${cpio}/bin/cpio -idmv
+
+
+      mkdir $out{,/share,/lib}
```
Comment (SuperSandro2000) [inline pkgs/misc/drivers/epkowa/default.nix:87]:
```suggestion
      mkdir -p $out/share $out/lib
```

**Review by SuperSandro2000 — COMMENTED** [review]
Just a minor nitpick. I wish you a merry Christmas, nek0!

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107584` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>epkowa</li>
  </ul>
</details>

File: pkgs/misc/drivers/epkowa/default.nix (hunk 1 above)
Comment (nek0) [reply pkgs/misc/drivers/epkowa/default.nix]: fixed in 70367f5bc9f. Merry christmas to you too.

**Comment by nek0** [comment]
Since this fact is not covered by the standard "what has been done" questions: The output of the package has been tested with said scanner and it's working.

**Pull request merged**